docs/a-tour-of-textile.md

```diff
@@ -1248,6 +1251,9 @@ Now that you have two peers running, invite the new account to your "My runs" th
     ok
     ```
 
+!!! Note
+    If the peer wasn't found, you may need to wait for a few seconds for the buddy textile instance to establish a peer connect to your primary textile instance.
```

Comment:
How about:
> You may have to make a couple attempts to send the invite because your second peer is probably not yet very well connected to the network.